src/SDK/Trace/ImmutableSpan.php

```diff
@@ -131,7 +132,7 @@ public function getEvents(): array
         return $this->events;
     }
 
-    public function getAttributes(): API\AttributesInterface
+    public function getAttributes(): AttributesInterface
```

Comment:
I'm a little confused about the mixed usage of `iterable` and `AttributesInterface` for attributes in different parts of the code. I just want to make sure that's intentional?

Reply:
Its intentional, API uses `iterable` (only parameters), SDK uses `AttributesInterface`.